lib/wavefront-metrics-formatter.js

```diff
@@ -3,60 +3,60 @@
+  points.push(pointLine(prefix, metricName, '.m1_rate', meter.oneMinuteRate(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.m5_rate', meter.fiveMinuteRate(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.m15_rate', meter.fifteenMinuteRate(), ts, tags));
   return points;
 }
 
-function timerPoints(timer, prefix, ts, tags) {
+function timerPoints(timer, metricName, prefix, ts, tags) {
   let points = [];
-  points.push(pointLine(prefix, timer.name, '.count', timer.count(), ts, tags));
-  points.push(pointLine(prefix, timer.name, '.mean_rate', timer.meanRate(), ts, tags));
-  points.push(pointLine(prefix, timer.name, '.m1_rate', timer.oneMinuteRate(), ts, tags));
-  points.push(pointLine(prefix, timer.name, '.m5_rate', timer.fiveMinuteRate(), ts, tags));
-  points.push(pointLine(prefix, timer.name, '.m15_rate', timer.fifteenMinuteRate(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.count', timer.count(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.mean_rate', timer.meanRate(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.m1_rate', timer.oneMinuteRate(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.m5_rate', timer.fiveMinuteRate(), ts, tags));
+  points.push(pointLine(prefix, metricName, '.m15_rate', timer.fifteenMinuteRate(), ts, tags));
   return points.concat(histoPoints(timer, prefix, ts, tags));
 }
 
-function histoPoints(histo, prefix, ts, tags) {
+function histoPoints(histo, metricName, prefix, ts, tags) {
   let points = [];
   const isHisto = histo instanceof Histogram;
   if (isHisto) {
     // send count if a histogram, otherwise assume this metric is being
     // printed as part of another (like a timer).
-    points.push(pointLine(prefix, histo.name, '.count', histo.count, ts, tags));
+    points.push(pointLine(prefix, metricName, '.count', histo.count, ts, tags));
   }
 
   let percentiles = histo.percentiles([.50,.75,.95,.98,.99,.999]);
-  points.push(pointLine(prefix, histo.name, '.min', isHisto? histo.min : histo.min(), ts, tags));
-  points.push(pointLine(prefix, histo.name, '.mean', histo.mean(), ts, tags));
-  points.push(pointLine(prefix, histo.name, '.max', isHisto ? histo.max: histo.max(), ts, tags));
-  points.push(pointLine(prefix, histo.name, '.stddev', histo.stdDev(), ts, tags));
-  points.push(pointLine(prefix, histo.name, '.p50', percentiles[.50], ts, tags));
-  points.push(pointLine(prefix, histo.name, '.p75', percentiles[.75], ts, tags));
-  points.push(pointLine(prefix, histo.name, '.p95', percentiles[.95], ts, tags));
-  points.push(pointLine(prefix, histo.name, '.p98', percentiles[.98], ts, tags));
-  points.push(pointLine(prefix, histo.name, '.p99', percentiles[.99], ts, tags));
-  points.push(pointLine(prefix, histo.name, '.p999', percentiles[.999], ts, tags));
+  points.push(pointLine(prefix, metricName, '.min', isHisto? histo.min : histo.min(), ts, tags));
```

Comment:
probably want to programmatically generate these

Reply:
Agree with most of the reformatting - lets take that up in a separate PR and not extend the scope of current PR.